clientd3d/d3drender_objects.c

```diff
@@ -0,0 +1,2411 @@
+// Meridian 59, Copyright 1994-2024 Andrew Kirmse and Chris Kirmse.
+// All rights reserved.
+//
+// This software is distributed under a license that is described in
+// the LICENSE file that accompanies it.
+//
+// Meridian is a registered trademark.
+#include "client.h"
+#include "d3drender_objects.h"
```

Comment:
Move to client.h.  This is done to get high-speed compilation with precompiled headers.